example/src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 
-import ReactTooltip from 'react-tooltip-rc';
+import ReactTooltip from 'react-tooltip';
```

Comment:
This repository is using `react-tooltip-rc` on NPM as it's name, this is only a fork of https://github.com/ReactTooltip/react-tooltip

Reply:
@pdeszynski I've updated the import back to `react-tooltip-rc`